src/components/ModelStateIndicator.tsx

```diff
@@ -1,10 +1,8 @@
 import { useEngineCommands } from './EngineCommands'
 import { Spinner } from './Spinner'
 import { CustomIcon } from './CustomIcon'
-
 export const ModelStateIndicator = () => {
   const [commands] = useEngineCommands()
-
   const lastCommandType = commands[commands.length - 1]?.type
```

Comment:
Nitpick: Were those blank lines added on purpose?


```suggestion
import { CustomIcon } from './CustomIcon'

export const ModelStateIndicator = () => {
  const [commands] = useEngineCommands()
  
  const lastCommandType = commands[commands.length - 1]?.type
```